package.json

```diff
@@ -42,7 +42,7 @@
     "json-loader": "^0.5.4",
     "json5": "^0.5.0",
     "jsx-control-statements": "3.1.5",
-    "leaflet": "0.7.3",
+    "leaflet": "1.0.3",
```

Comment:
Unfortunately I do. They completely changed feature selection behaviour between Leaflet 1.0.1 and 1.0.3 (both stable, same minor version releases), and I back-ported the 1.0.1 behaviour to 1.0.3, which is probably fragile to any further changes.